atlp-rwanda/atlp-pulse-fn
# Ft reply on remaks

# PR Description
1. Reply on remark performance by trainee
Please include a brief description of the pull request you want to raise.
. A trainee is able to reply on remarks he/she recceived on a performance rating



# How has this been tested?

1. Clone This repo
2. connect with your local database with mongoose credintials
3. run `npm i` to install your dependendecies
4. run `npm run dev` to test your development application
5. run your backend to check the functionalities locally


# Screenshots (If appropriate)

<img width="630" alt="Screen Shot 2022-10-19 at 17 04 42" src="https://user-images.githubusercontent.com/100874204/196729937-f1e0ab25-d755-4ae4-864c-e78cd665722c.png">
